server/test/uk/gov/ons/addressIndex/server/modules/ElasticsearchRepositorySpec.scala

```diff
@@ -1160,7 +1160,7 @@ class ElasticsearchRepositorySpec extends WordSpec with SearchMatchers with Elas
                                         },
                                         {  
                                            "dis_max":{
-                                              "tie_breaker":${queryParams.disMaxTieBreaker},
+                                              "tie_breaker":${queryParams.excludingDisMaxTieBreaker},
```

Comment:
It would be covered by the TotalQuery test that is now ignored (we decided to ignore it until the query is stable, if I may remind you)